content/200-concepts/100-components/02-prisma-client/053-client-extensions/130-result.mdx

```diff
@@ -26,7 +26,7 @@ Use the `result` extension component to add custom fields and methods to query r
 To add a custom [field](#add-a-custom-field-to-query-results) or [method](#add-a-custom-method-to-the-result-object) to query results, use the following structure. In this example, we add the custom field `myComputedField` to the result of a `user` model query.
```

Comment:
@ruheni I think we should ad a similar admonition to this file

Reply:
We could do without the admonition here because the extension modifies the results of a query and but not the query sent to the database 😄

Reply:
Got some feedback from Serhii ([Internal conversation](https://prisma-company.slack.com/archives/C043X6JBND6/p1693828548570949?thread_ts=1693825660.132309&cid=C043X6JBND6))

>My opinion, pretty much for all kinds of extensions:
you need to store primary instance in a separate variable if you want to use it later (without the extension) or you want to use independent instances of extensions with different settings within your app (for example, RLS extension would need session information and can not be applied globally)
you are fine to use inline version if you won’t need primary instance for anything else: most result, client and model extensions would fall into this category but there could be exceptions.

TODO: 

- [ ] Update the `query` extension docs, removing instances of `xprisma`

- [ ] I'll create a separate issue for us to document the difference between the two — primary vs "variant"